submit-api/src/submit_api/models/update_request.py

```diff
@@ -25,6 +25,7 @@ class UpdateRequestStatus(enum.Enum):
     OPEN = 'OPEN'
```

Comment:
I'm thinking now or later maybe we can change this status to something like "CREATED" which is probably more accurate for our use case.

Reply:
not a blocker for merge, this could definitely be done in a separate task